avocado/core/multiplexer.py

```diff
@@ -397,11 +397,8 @@ def parse(self, args):
 
```

Comment:
This commit is a prep work to get to the point where we can follow the diagram. What the commit message tries to say is that with this change the `core` API should have no knowledge about producing the variants (only the `Mux` allows to get variants from the `yaml_to_mux`. I'll try to improve the message, though... The same with avocado-virt. I used commit messages to store my ideas, before the final release I have to adjust them all.